src/main/java/menelaus/view/game/WinScreen.java

```diff
@@ -1,20 +1,14 @@
 package menelaus.view.game;
 
-import java.awt.BorderLayout;
-import java.awt.EventQueue;
+import menelaus.view.KabaSujiJPanel;
 
-import javax.swing.JFrame;
-import javax.swing.JPanel;
-import javax.swing.border.EmptyBorder;
-import javax.swing.GroupLayout;
+import javax.swing.*;
 import javax.swing.GroupLayout.Alignment;
-import javax.swing.JLabel;
-import java.awt.Font;
-import javax.swing.JButton;
+import java.awt.*;
 
 public class WinScreen extends JFrame {
 
-	private JPanel contentPane;
+	private KabaSujiJPanel contentPane;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'contentPane' could be replaced by a local variable.](https://www.codacy.com/app/dmvouldjeff/menelaus/file/2417307594/issues/source?bid=3228196&fileBranchId=3236530#l11)